docs/topics/readme/cd-rest-http-secured-thorntail.properties

```diff
@@ -0,0 +1,6 @@
+localRunCMD=mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}')
+app-name=thorntail
```

Comment:
??

Reply:
It's used here: https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/readme/rest-http-secured-README.adoc#interacting-with-the-booster-on-a-single-node-openshift-cluster